editor/filesystem_dock.cpp

```diff
@@ -72,6 +72,10 @@ Control *FileSystemTree::make_custom_tooltip(const String &p_text) const {
 	return FileSystemDock::get_singleton()->create_tooltip_for_path(item->get_metadata(0));
 }
 
+FileSystemTree::FileSystemTree() {
+	set_auto_translate(false);
```

Comment:
This method is deprecated 🙃
Also you don't need to create a constructor to set it.

Reply:
Brain fart. :stuck_out_tongue_closed_eyes:
But, where else do I set it? Unless you mean setting it outside the class by its parent, which feels a wee-bit hacky.

Reply:
> Unless you mean setting it outside the class by its parent

Yes, that's how auto-translation is disabled everywhere else. The fact that FileSystemTree is a custom class does not make it suddenly a bad solution.

Reply:
I disagree a little, but it's not a hill worth dying for, change made.

Reply:
I did something similar in the constructor of `FileSystemList` (#88716)

Most controls are translating by default because they are for general use. For a control that displays filenames, it makes sense to default to not translating.

Reply:
FileSystemList already had a constructor.
Either way, if there is a single instance of the class, it doesn't matter.